feel/feelalg/vectorpetsc.hpp

```diff
@@ -170,6 +171,23 @@ class VectorPetsc : public Vector<T>
     }
 
     /**
+     * @brief creates a VectorPetsc out of a VectorUblas
+     *
+     * there is no copy, PETSc will use the storage of the VectorUblas
+     */
+    template<typename Storage>
+    VectorPetsc( VectorUblas<T,Storage>& v )
+        :
+        super( v.mapPtr() ),
+        M_destroy_vec_on_exit( false )
+        {
+            VecCreateMPIWithArray( v.map().worldComm().globalComm(),
+                                   1, v.map().nLocalDofWithoutGhost(), v.map().nDof(),
+                                   std::addressof( v[0] ), &M_vec );
+            this->M_is_initialized = true;
```

Comment:
@prudhomm I think it's not good in parallel. the size of M_vec is nLocalDofWithoutGhost() but v is nLocalDofWithGhost(). And the ordering is not the same.